src/test/scala/labs/Lab1Test.scala

```diff
@@ -96,33 +119,14 @@ class SingleCycleAddTesterLab1 extends CPUFlatSpec {
 class SingleCycleRTypeTesterLab1 extends CPUFlatSpec {
   behavior of "Single Cycle CPU"
   for (test <- InstTests.rtype) {
-    if (test.binary !== "add0") {
+    if (test.binary != "add0" && test.binary != "add1" && test.binary != "add2") {
       it should s"run R-type instruction ${test.binary}${test.extraName}" in {
         CPUTesterDriver(test, "single-cycle") should be(true)
       }
     }
   }
 }
 
-
```

Comment:
Did this accidentally get deleted?

Reply:
Oops, sorry Jason, my reply ended up being a review instead of a comment so it didn't post. It wasn't deleted, I just moved the add0 test above the all r-type test to better reflect the test order. Just in case, I tested it again and it looked good.